bakery/src/task.js

```diff
@@ -9,7 +9,7 @@ module.exports.command = commandUsage
 module.exports.aliases = ['t']
 module.exports.describe = 'builds a bakery pipeline task runnable with fly execute'
 module.exports.builder = yargs => {
-  yargs.usage(`Usage: ${process.env.CALLER || 'build.js'} ${commandUsage}`)
+  yargs.usage(`Usage: ${process.env.CALLER || 'build'} ${commandUsage}`)
```

Comment:
Just curious: I believe this to handle where `process.env.CALLER` doesn't have a value and we want to display `build` vs `build.js` in the output. If so, how are you invoking it to get the `||` path?

Reply:
I have not been able to invoke it to get the `||` path. But it looks like I may have been confused because `build.js` doesn't do anything..?

```bash
(cops) [Tue Jun 16 15:15:01]brenda@brenda-openstax:bakery[bn/cnx#929/production-ready-details !?] $ build.js task --help
-bash: build.js: command not found
(cops) error 127 [Tue Jun 16 15:16:46]brenda@brenda-openstax:bakery[bn/cnx#929/production-ready-details !?] $ ./build task --help
Usage: ./build task <taskname> [options]...
...
```

Where does the display of `build` vs `build.js` happen?

Reply:
I put `.js` back. I don't clearly remember why I removed it, but I still don't understand it.